hms-api/src/repository/idea-repository.ts

```diff
@@ -44,6 +44,21 @@ export async function listIdeasForHackathon(
   );
 }
 
+export async function listIdeasAll(): Promise<Idea[]> {
```

Comment:
this is a VERY risky thing to do... eventually this will become VERY large, in fact, dynamo won't return all of them.  It will return a paged response that you would need to iterate over.  That is actually true for basically all of the requests we are making, but when you limit it based on some query parameter it becomes less of a concern